lmcache/config.py

```diff
@@ -53,6 +54,7 @@ def from_defaults(
         chunk_size: int = 256,
         local_device: str = "cuda",
         max_local_cache_size: int = 5,
+        disk_url: str = "",
```

Comment:
A few quick comments about `disk_url` here.
- The name of `disk_url` itself is a bit ambiguous. Is it the URL of the disk manager process? Then maybe we should use something like `disk_manager_url`
- If it's an optional configuration, it's clearer to use `None` as the default value, instead of `""`
- In the `from_file` code, `disk_url` is first set by `disk_url = config.get("disk_url", None)`, and then be changed again when `if re.match(r"disk_url://(.*)", local_device)`. This is pretty confusing. Why changing the value of `local_device` can overwrite the `disk_url`? Does this mean `disk_url` in the yaml config is useless?

Reply:
Actually, the `disk_url`  in yaml config means `disk folder path` in disk_address_manager and `address manager url` in LMCache backend.